src/MICore/LaunchOptions.cs

```diff
@@ -457,7 +455,7 @@ static internal LocalLaunchOptions CreateFromJson(JObject parsedOptions)
 
             if (launchOptions == null)
             {
-                throw new InvalidLaunchOptionsException(MICoreResources.Error_InvalidLaunchOptions);
+                throw new InvalidLaunchOptionsException(MICoreResources.Error_LaunchOptionsNull);
```

Comment:
@gregg-miskelly I changed it to the message that says we weren't able to serialize it. It happens when we can't serialize it as either launch or attach options.